project-oak/oak
# Run runner cargo commands only for modified crates

Allows running the following commands only for the recently modified files and crates affected by them: 
- run-cargo-tests
- run-examples
- run-functions-examples
- format
- check-format

To use this feature, the `--commits=N` should be passed to the command, where `N>0` is the number of past commits to be considered for finding the modified files (`git diff HEAD~N` is used for finding the modified files). For this to work for new files, the file should be added to the git index.

For now this is intended to be used when running commands locally.
